docs/releasenotes/0.23.md

```diff
@@ -0,0 +1,34 @@
+# actions-runner-controller v0.23.0
+
+https://github.com/actions-runner-controller/actions-runner-controller/milestone/3
+
+# BREAKING CHANGE : Workflow job webhooks require an explicit field set
+
+Previously the webhook workflow job was set as the default if no `githubEvent` was set:
+
+```yaml
+  scaleUpTriggers:
+  - githubEvent: {}
+    duration: "30m"
+```
+
+You now need to set `workflowJob: {}` explicitly
+
+```yaml
+  scaleUpTriggers:
+  - githubEvent:
+      workflowJob: {}
+    duration: "30m"
+```
+
+# BREAKING CHANGE : topologySpreadConstraints renamed to topologySpreadConstraint
+
+Previously to use the pod `topologySpreadConstraint:` attribute in your runners you had to set `topologySpreadConstraints:` instead, this was a typo and has been corrected.
+
+# BREAKING CHANGE : Default sync period is now 1 minute instead of 10 minutes
+
+Since caching as been implemented the default sync period of 10 minutes is unnecessarily conservative and gives a poor out of the box user experience. If you need a 10 minute sync period ensure you explicitly set this value.
+
+# BREAKING CHANGE : A metric is set by default
+
+Previously is no metric was provided and you were using pull based scaling, `TotalNumberOfQueuedAndInProgressWorkflowRuns` was applied. No default is set now and the end user must always set this metric explicitly if they want to use it.
```

Comment:
Worth adding an example manifest?

```yaml
apiVersion: actions.summerwind.dev/v1alpha1
kind: HorizontalRunnerAutoscaler
metadata:
  name: example
spec:
  #snip
  metrics:
  - type: TotalNumberOfQueuedAndInProgressWorkflowRuns
```

Reply:
Yup, I'll add a **Migration Steps** section for each, just throwing stuff up atm.